pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -135,6 +135,7 @@ func (c *Cluster) needHTTPBindFix() bool {
 // always do this and (b) make sure that all forms of the configuration option
 // are removed (see the init container factory method). Once the minimum
 // supported version of Rook contains the fix, all of this can be removed.
+// #nosec
```

Comment:
Ditto on document `#nosec`

Reply:
It seems dangerous to disable security checking for an entire method that does as much as this one does.